apps/core/scenes/results/Results.js

```diff
@@ -56,6 +56,9 @@ type Props = {|
   +nightsInDestinationTo: string,
   +isNightsInDestinationSelected: boolean,
   +setBookingToken: string => void,
+  +routerQuery: SearchParameters,
+  +setStateFromQueryParams: SearchParameters => void,
```

Comment:
The `routerQuery` and `setStateFromQueryParams` do not use anywhere, I guess.

Reply:
Hmm its not, probably forgot to remove as I dont remember already reason behind putting there in a first place.
Thanks! Removing